go-controller/pkg/ovn/endpoints_test.go

```diff
@@ -111,43 +90,33 @@ var _ = Describe("OVN Namespace Operations", func() {
 
 				endpointsT := *newEndpoints("endpoint-service1", "namespace1",
 					[]v1.EndpointAddress{
-						v1.EndpointAddress{
+						{
 							IP: "10.125.0.2",
 						},
 					},
 					[]v1.EndpointPort{
-						v1.EndpointPort{
+						{
 							Name:     "portTcp1",
 							Port:     8080,
 							Protocol: v1.ProtocolTCP,
 						},
-						v1.EndpointPort{
```

Comment:
@alexanderConstantinescu any particular reason these EndpointPorts are getting removed? Not necessary?